legal-api/src/legal_api/services/filings/validations/registration.py

```diff
@@ -41,13 +45,25 @@ def validate(registration_json: Dict) -> Optional[Error]:
 
     legal_type_path = '/filing/registration/nameRequest/legalType'
     legal_type = get_str(registration_json, legal_type_path)
+    filing_type = 'registration'
     if legal_type not in [Business.LegalTypes.SOLE_PROP.value, Business.LegalTypes.PARTNERSHIP.value]:
         return Error(
             HTTPStatus.BAD_REQUEST,
             [{'error': babel('A valid legalType for registration is required.'), 'path': legal_type_path}]
         )
+    if not validate_certify_name(registration_json):
+        required_permission = ListActionsPermissionsAllowed.EDITABLE_COMPLETING_PARTY.value
+        message = f'Permission Denied - You do not have permissions to change completing party in this filing.'
+        error = PermissionService.check_user_permission(required_permission, message)
+        if error:
+            return error
+    if validate_document_delivery_completing_party(registration_json) or validate_editable_completing_party(registration_json, filing_type):
+        required_permission = ListActionsPermissionsAllowed.EDITABLE_COMPLETING_PARTY.value
+        message = f'Permission Denied - You do not have permissions to change certified by in this filing.'
+        error = PermissionService.check_user_permission(required_permission, message)
+        if error:
+            return error
```

Comment:
move `validate_certify_name` and `validate_document_delivery_completing_party` to `validation.py` (next to `validate_staff_payment`), to avoid duplicate checks

Note: Any prop in header can be moved to `validation.py`